### PR TITLE
Added annotation for datalens

### PR DESCRIPTION
**Pull request by JoshuaOliphant**

### CODE REVIEW

File: kubelogin/templates/deployment.yaml [hunk 1]
```diff
@@ -19,6 +19,8 @@ spec:
     metadata:
       labels:
         app: {{ template "kubelogin.fullname" . }}
+      annotations: 
+        datalens.nordstrom.com/log-stream-name: {{.Values.kubelogin.datalens-annotations}}
```
Comment (jkinkead) [inline kubelogin/templates/deployment.yaml]: Shouldn't we be avoiding internal-only features?

Comment (ashish-amarnath) [reply kubelogin/templates/deployment.yaml]: @JoshuaOliphant  Please refer to `service.yaml` for an example to allow us to supply custom annotation on the pod. You'll want to do something like this.
```
{{- if .Values.kubelogin.pod.annotations }}
  annotations:
{{ toYaml .Values.kubelogin.pod.annotations | indent 4 }}
{{- end }}
```

Comment (ashish-amarnath) [reply kubelogin/templates/deployment.yaml]: you'll want this
```
{{- if .Values.kubelogin.pod.annotations }}
      annotations:
{{ toYaml .Values.kubelogin.pod.annotations | indent 8 }}
{{- end }}
```
not `indent 4`

**Comment by mariusgrigoriu** [comment]
Sorry but this is a Nordstrom specific change and does not belong in the open source repository. This should be applied to our internal repository.

**Comment by ashish-amarnath** [comment]
LGTM.

**Pull request merged**